pkg/apis/autoscaling/annotation_validation.go

```diff
@@ -275,3 +276,23 @@ func validateInitialScale(config *autoscalerconfig.Config, m map[string]string)
 	}
 	return nil
 }
+
+
+func validateMaxInitialScale(config *autoscalerconfig.Config, m map[string]string) *apis.FieldError{
+
+	max,errs := getIntGE0(m,MaxScaleAnnotation)
```

Comment:
These annotations could be omitted from the revision so you would need to handle that

see https://github.com/knative/serving/blob/aef6c59c424a512d745c13b16d168dec795faddf/pkg/apis/autoscaling/annotation_validation.go#L267